ambry-rest/src/test/java/com.github.ambry.rest/RestTestUtils.java

```diff
@@ -51,6 +55,22 @@ public static HttpRequest createRequest(HttpMethod httpMethod, String uri, HttpH
     return httpRequest;
   }
 
+  /**
+   * Creates a {@link FullHttpRequest} with the given parameters.
+   * @param httpMethod the {@link HttpMethod} required.
+   * @param uri the URI to hit.
+   * @param content the content to add to the request
+   * @return a {@link FullHttpRequest} with the given parameters.
+   */
+  public static FullHttpRequest createRequest(HttpMethod httpMethod, String uri, HttpHeaders headers, byte[] content) {
```

Comment:
Is it better to call the method `createFullRequest` to distinguish it from another `createRequest` method in this file?

Reply:
Done